tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot

```diff
@@ -306,3 +308,38 @@ Re-validate License For Disabled Application From Enabled Page
    Wait Until Page Contains   To remove card click
    ${buttons_here}=  Get WebElements    xpath://div[contains(@class,'popover__body')]//button[text()='here']
    Click Element  ${buttons_here}[0]
+
+Check CSS Property Has The Expected Value
```

Comment:
I think this keyword should also be in tests/Resources/Common.robot

It could be renamed to `Verify CSS Property Value` or `CSS Property Value Should Be` to match the naming of other existing keywords

Reply:
I like this `CSS Property Value Should Be`, it is way better. I'll move it to common.robot. Thank you!

Reply:
done